ktor-shared/ktor-sse/common/src/io/ktor/sse/ServerSentEvent.kt

```diff
@@ -15,16 +15,22 @@ import io.ktor.utils.io.*
  *  @property retry reconnection time, in milliseconds to wait before reconnecting.
  *  @property comments comment lines starting with a ':' character.
  */
-public class ServerSentEvent(
-    public val data: String? = null,
+public data class ServerSentEvent<T>(
+    public val data: T? = null,
```

Comment:
Should we also split this class in 2 so String would be default?

Reply:
I split into `ServerSentEven`t and `ParameterizedServerSentEvent<T>`